packages/components/src/components/action-bar/action-bar.browser.e2e.tsx

```diff
@@ -107,4 +112,107 @@ describe("calcite-action-bar", () => {
       "calcite-action-group",
     );
   });
+
+  describe("keyboard navigation and selection-mode", () => {
+    it("supports toolbar pattern keyboard navigation", async () => {
+      const { el } = await mount<"calcite-action-bar">(
+        <calcite-action-bar
+          floating
+          layout="horizontal"
+          overflow-actions-disabled
+          selection-appearance="neutral"
```

Comment:
To keep tests minimal and simple, let's avoid setting attributes unrelated to the test or setting them to the default (e.g., `neutral` is the default selection appearance).